No matching version found for @mbunifu/library@^0.1.0

```
» npm i
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @mbunifu/library@^0.1.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'derp'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dwade/.npm/_logs/2018-12-09T00_51_24_031Z-debug.log
```

--- a/templates/generator/package.json
+++ b/templates/generator/package.json
@@ -5,7 +5,7 @@
   "version": "0.0.1",
   "description": "Generator for projects.",
   "dependencies": {
-    "@mbunifu/library": "^0.1.0",
+    "@mbunifu/library": "^0.0.1",
     "inquirer": "^5.2.0"
   },
   "scripts": {

